### PR TITLE
Adds crypto fees revenue command

### PR DESCRIPTION
**Pull request by jose-donato**
# Description

Integrates crypto fees revenue adapter from [@CryptoStats_](https://twitter.com/CryptoStats_). There are many other adapters (FREE) that we can integrate from cryptostats

Still need to:
* add tests
* add to sdk

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/defi/llama_model.py [hunk 1]
```diff
@@ -30,6 +30,29 @@
...
+    -------
+    pd.DataFrame
+        Information about chains
+    """
+    response = request(API_URL + "/chains")
+    if response.status_code != 200:
+        raise Exception(f"Status code: {response.status_code}. Reason: {response.text}")
+    try:
+        df = pd.DataFrame(response.json())
+        df.replace({float(np.nan): None}, inplace=True)
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/defi/llama_model.py:48]: inplace=True is bad,  You should also use fillna

File: openbb_terminal/cryptocurrency/discovery/cryptostats_view.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        title="Crypto Fees",
+        limit=limit,
+    )
+
+    export_data(
+        export,
+        os.path.dirname(os.path.abspath(__file__)),
+        "fees",
+        df,
+    )
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/discovery/cryptostats_view.py:69]: sheet name argument

File: openbb_terminal/cryptocurrency/discovery/discovery_controller.py [hunk 3]
```diff
@@ -88,8 +92,86 @@ def print_help(self):
...
+            prog="fees",
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            description="""
+            Cryptos where users pay most fees on [Source: CryptoStats]
+            """,
+        )
+
+        parser.add_argument(
+            "-mc",
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/discovery/discovery_controller.py:112]: we should not have 2 letters on single dash argument

File: openbb_terminal/cryptocurrency/discovery/discovery_controller.py [hunk 4]
```diff
@@ -88,8 +92,86 @@ def print_help(self):
...
+            "--marketcap",
+            action="store_true",
+            dest="marketcap",
+            default=False,
+            help="Include the market cap rank",
+        )
+        parser.add_argument(
+            "-tvl",
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/discovery/discovery_controller.py:120]: same

**Review by jmaslek — COMMENTED** [review]
fix the source and then its picasso level beauty

**Review by jmaslek — APPROVED** [review]

**Comment by jose-donato** [comment]
@jmaslek fixed the tests
failing on a stocks on `ERROR tests/openbb_terminal/stocks/fundamental_analysis/test_yahoo_finance_view.py::test_display_splits[display_splits-get_splits]`

**Pull request merged**